npm/vue2/src/index.ts

```diff
@@ -290,6 +292,30 @@ const injectStyles = (options: StyleOptions) => {
   return injectStylesBeforeElement(options, document, getContainerEl())
 }
 
+/**
+ * Extract the component name from the object passed to mount
+ * @param componentOptions the compoennt passed to mount
+ * @returns name of the component
+ */
+function getComponentDisplayName (componentOptions: any): string {
+  if (componentOptions.name) {
+    return componentOptions.name
+  }
+
+  if (componentOptions.__file) {
+    const filepathSplit = componentOptions.__file.split('/')
+    const fileName = filepathSplit[filepathSplit.length - 1]
+
+    // remove the extension .js, .ts or .vue from the filename to get the name of the component
+    const baseFileName = fileName.replace(/\.(js|ts|vue)?$/, '')
```

Comment:
If `filepathSplit` does not have a `/`, `fileName` will be `undefined`, then we will get "Cannot call replace on undefined" here.

I think we need

```js
const fileName = filepathSplit[filepathSplit.length - 1] ?? DEFAULT_COMP_NAME
```